Users/coala_as_Git_Hook.rst

```diff
@@ -1,7 +1,7 @@
 ﻿Git Hooks
 =========
```

Comment:
first of all, i believe its .rst, not .html (in commits hortlog).

also, what is that  body wrapping? text should be wrapped @72 chars, not at 20! it looks really bad.